hardhat.config.ts

```diff
@@ -278,6 +287,10 @@ const config: HardhatUserConfig = {
       url: "https://scroll-sepolia.drpc.org",
       accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
     },
+    optimism_sepolia_testnet:{
```

Comment:
All good, but I don't think this line here is formatted correctly. the colon should have spaces around